azext_iot/central/_help.py

```diff
@@ -410,10 +413,13 @@ def _load_central_monitors_help():
         "iot central diagnostics monitor-events"
     ] = """
         type: command
-        short-summary: Monitor device telemetry & messages sent to the IoT Hub for an IoT Central app.
+        short-summary: View device telemetry messages sent to the IoT Central app.
```

Comment:
We might want some indication that this means data made it to the entrance of Central, but doesn't necessarily mean it made it all the way through the system

e.g. if they see it here, but not on raw data view/their tiles, they might need to file a ticket against Central

Reply:
does the telemetry send to this device will not be going through Central? I feel if they can see from here, they should be able to see from tile as well, right?

Reply:
Chatted offline, might be worth filing a bug on the backlog to look at this once people are back. I think one key thing to document here is that if data appears in monitor events/validate-messages, but does not appear in central, the customer should file a CSS ticket